dmaws/context.py

```diff
@@ -19,6 +20,18 @@ def __init__(self, home=os.getcwd(), dry_run=False):
         self.stacks = {}
         self.create_dependencies = False
 
+    def new_context(self, stage, environment, vars_files):
+        """Create a new context with different variables"""
+        ctx = Context()
+        ctx.stage = stage
+        ctx.environment = environment
+        ctx.load_variables(
+            files=ctx.get_variables_files(True, vars_files))
+        ctx.stacks = copy.deepcopy(self.stacks)
```

Comment:
The reason we don't do a fresh `ctx.load_stacks(stacks_file)` is that we don't have the `stacks_file` at the point that this is used (in the `migratedata` command). The reason we need a new context is to create and delete stacks in two different environments at the same time. 

If you think might be a risk it may be worth while just biting the bullet and getting all the construction code out of `dmaws.cli.py`. What do you think?